.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+- '2.7'
+- '3.4'
+- '3.5'
+- '3.6'
```

Comment:
We don't require any specific python version here. Use latest version or maybe go with the default available version in travis. Multiple versions makes sense when you need to test python code. For a theme, I think it's irrelevant. Multiple versions would just increase build times here.